WooCommerce/WooCommerceTests/ViewRelated/PaymentTableViewCellTests.swift

```diff
@@ -0,0 +1,95 @@
+import XCTest
+@testable import WooCommerce
+@testable import Networking
+
+final class PaymentTableViewCellTests: XCTestCase {
+    private var cell: PaymentTableViewCell?
+    private var viewModel: OrderPaymentDetailsViewModel?
+    private var order: Order?
```

Comment:
(same non-blocking suggestion as before to make these variables that are always set before each test setup non-optional (using ❗️ always gives me chills, but many ❓ are also a bit overwhelming for me)